Products/ZenUtils/zenpack.py

```diff
@@ -481,6 +481,10 @@ def doRestore(pack):
         fixedSomething = len(sortedPacks) > 0
         while len(sortedPacks) > 0:
             packListLen = len(sortedPacks)
+            # Load zenpacks files before normal installation to avoid
+            # upgrade breaking (ZEN-24274).
+            for packs in sortedPacks:
+                self._loadFiles(packs, zpsToRestore[packs][0])
```

Comment:
I was confused a bit by `packs`.  It's actually a `pack` (singular).  So a variable rename is in order here.